.prow/features.yaml

```diff
@@ -272,46 +272,3 @@ presubmits:
               cpu: 2
             limits:
               memory: 6Gi
-
-  - name: pre-kubermatic-legacy-machine-controller-user-data-e2e
-    decorate: true
-    clone_uri: "ssh://git@github.com/kubermatic/kubermatic.git"
-    labels:
-      preset-aws-e2e-kkp: "true"
```

Comment:
Still relevant since we haven't removed the MC user-data plugins yet.

Reply:
So should this be bumped to another Kube version? Cause KKP won't be able to crate 1.24 cluster anymore..

Reply:
Yup, that would make sense, ideally k8s 1.27.